packages/client/src/Components/PaperEdits/PaperCutsPlayer/ProgressBar.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable template-curly-spacing */
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+// import './styles.css';
+
+class PaperCutsPlayerProgressBar extends React.PureComponent {
+  constructor(props) {
+    const {
+      width = 640,
```

Comment:
Same thing here, if the width could be responsible, I think it would make the component more flexible

Reply:
The width is responsive, this is just a default value. You can set the width whereever you use the component.